frontend/src/domain/Task.ts

```diff
@@ -39,7 +39,7 @@ export const TaskIntent = z
   .object({
     userId: z.number(),
     projectId: z.string().min(1, { message: 'Project is required' }),
-    taskType: z.string().optional(),
+    taskType: z.string().nullable(),
     story: z.string(),
```

Comment:
The story and description will also be nullable. I am mentioning in case it is something you need to account for.